cmd/internal/ecs/add.go

```diff
@@ -533,7 +533,7 @@ func getTaskState(wf *AddWorkflow) (nextState optionals.Optional[AddWorkflowStat
 	// Check that the postman-insights-agent is not already present
 	for _, container := range output.ContainerDefinitions {
 		image := aws.ToString(container.Image)
-		if matchesImage(image, postmanECRImage) || matchesImage(image, akitaECRImage) || matchesImage(image, akitaDockerImage) {
+		if matchesImage(image, postmanECRImage) {
```

Comment:
I think we should probably continue looking for the old docker images?  We don't want users who convert over to Postman to have two copies of the agent running.

Reply:
In b4a9481.